service/worker/workerdeployment/workflow_test.go

```diff
@@ -47,9 +56,239 @@ func (s *WorkerDeploymentSuite) SetupTest() {
 	s.controller = gomock.NewController(s.T())
 	s.env = s.WorkflowTestSuite.NewTestWorkflowEnvironment()
 	s.env.RegisterWorkflow(Workflow)
+	s.env.SetWorkflowRunTimeout(30 * time.Second)
 }
 
 func (s *WorkerDeploymentSuite) TearDownTest() {
 	s.controller.Finish()
 	s.env.AssertExpectations(s.T())
 }
+
+// To incorporate testVars in our tests.
+func (s *WorkerDeploymentSuite) Name() string {
+	fullName := s.T().Name()
+	if len(fullName) <= 30 {
+		return fullName
+	}
+	short := fmt.Sprintf("%s-%08x",
+		fullName[len(fullName)-21:],
+		farm.Fingerprint32([]byte(fullName)),
+	)
+	return strings.Replace(short, ".", "|", -1)
```

Comment:
You can use `[ReplaceAll](https://cs.opensource.google/go/go/+/go1.24.1:src/strings/strings.go;l=1180) [¶](https://pkg.go.dev/strings#ReplaceAll) for this instead.